### ISSUE TITLE
Stimulants dependence

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6535]
**Preferred term label:** Stimulants dependence

**Synonyms**
Stimulants addiction 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Addictive behavior HP:0030858

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
I do not think the addiction terms are phenotypic features, but they would fit in this branch, 
https://hpo.jax.org/app/browse/term/HP:0032443
what do you think?

**Comment by pnrobinson** [comment]
See here fore some definitions https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5525418/

**Comment by pnrobinson** [comment]
added
